src/link/rgblink.1

```diff
@@ -54,6 +55,12 @@ Write a tiny
 ROM file.
 This forces all ROMX sections to be of type ROM0, and increases the ROM0
 section size from 16KiB to 32KiB.
+.It Fl l Ar linkerscript
+Specify a linkerscript file that tells the linker how sections must be placed in
+the ROM. This file has priority over the attributes assigned in the source code,
```

Comment:
A new sentence starts on a new line. This is how troff knows to apply inter-sentence spacing rules.